extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -165,9 +176,10 @@ export class CookieWriter {
    * Expand the value and write to cookie if necessary
    * @param {string} cookieName
    * @param {string} cookieValue
+   * @param {number} cookieExpireDateMs
```

Comment:
nit: looks like the value could be undefined. Maybe 
`@param {number=} opt_cookieExpireDateMs`?